packages/fxa-auth-server/lib/senders/emails/partials/button/index.scss

```diff
@@ -0,0 +1,28 @@
+@use '../../global.scss';
+
+@mixin text-lg {
+  font-size: 18px !important;
+  line-height: 28px !important;
+}
+$blue-background: #0a84ff !important;
```

Comment:
Let's call this `$blue-400` since it's a color and not necessarily a background color. `400` is the closest value this hex code is to what's in the `fxa-settings` design guide (500 looks much darker to me).

Let's also use the `blue-400` Tailwind color from our design guide which is `#0090ED`.

Our design team will be fine with small tweaks like this for consistency across FxA. 🙂